portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -224,7 +224,9 @@ definition {
 			RobotType.robotTypeShortcut(locator1 = "Ctrl + v");
 		}
 
-		Pause(locator1 = "5000");
+		AssertTextEquals(
+			locator1 = "WCEditWebContent#TEXT_HEADER",
+			value1 = "Sample document for editor area test");
```

Comment:
There's no need to target a header here specifically. I suggest use something like waitForTextPresent instead of assert on CKEDITOR_CONTENT where the path contains the text.